tests/snippets/builtin_dict.py

```diff
@@ -1,3 +1,5 @@
+import pytest
```

Comment:
Unfortunately, pytest is running the test but not available from the test code. See `testutils.py`. it includes `assert_raises` function which almost working like pytest.raises

Reply:
Could you tell me how to cPython test in local?

Reply:
README describes how to run test in local.
Roughly, running a single file is: `python tests/snippets/builtin_dict.py`